.github/pull_request_template.md

```diff
@@ -44,10 +46,12 @@ your domain(s) directly with that third-party, and it is inappropriate
 to submit entries to the PSL as a means to work around those limits or 
 restrictions.
 -->
-  * [ ] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)
-    - [Cloudflare](https://developers.cloudflare.com/learning-paths/get-started/add-domain-to-cf/add-site/)
-    - [Letsencrypt](https://letsencrypt.org/docs/rate-limits/)
-    - <UPDATE THIS LIST WITH YOUR LIMITATIONS! REMOVE ENTRIES WHICH DO NOT APPLY! REMOVE THIS LINE!>
+
+<!-- MAKE SURE UPDATE THE FOLLOWING LIST WITH YOUR LIMITATIONS! REMOVE ENTRIES WHICH DO NOT APPLY! -->
```

Comment:
This was intentionally part of the list, to make it very obvious when people haven't at least modified the list to remove it.